tools/e2etesting/NestedEdge/scripts/import_acr.sh

```diff
@@ -88,13 +88,14 @@ az acr import --name $acrName --force --source mcr.microsoft.com/azureiotedge-hu
 echo "diagnostics..."
 az acr import --name $acrName --force --source mcr.microsoft.com/azureiotedge-diagnostics:1.2.0-rc4 --image azureiotedge-diagnostics:1.2.0-rc4
 echo "monitor..."
-az acr import --name acrqq4mbi.azurecr.io --force --source mcr.microsoft.com/azuremonitor/containerinsights/ciprod:iot-0.1.3.3 --image ciprod:latest
+az acr import --name $acrName --force --source mcr.microsoft.com/azuremonitor/containerinsights/ciprod:iot-0.1.3.3 --image ciprod:latest
 echo "API proxy..."
 az acr import --name $acrName --force --source mcr.microsoft.com/azureiotedge-api-proxy:latest --image azureiotedge-api-proxy:latest
 echo "IIoT - OpcPlc server ..."
 az acr import --name $acrName --force --source mcr.microsoft.com/iotedge/opc-plc:latest --image opc-plc:latest
 echo "IoT - OPC Publisher..."
 az acr import --name $acrName --force --source mcr.microsoft.com/iotedge/opc-publisher:2.7.206 --image opc-publisher:2.7.206
+az acr import --name $acrName --force --source mcr.microsoft.com/iotedge/opc-publisher:2.7.206 --image iotedge/opc-publisher:2.7.206
```

Comment:
this looks loke a duplicate, any particular reason for it?

Reply:
In the test code for standalone we look for the image "iotedge/opc-publisher:2.7.206" instead of "opc-publisher:2.7.206" and it fails. I tried not to change too much xunit code, but scripts instead

Reply:
great!

---
In reply to: [613041587](https://github.com/Azure/Industrial-IoT/pull/1099#discussion_r613041587) [](ancestors = 613041587)